--- /dev/null	2013-07-17 20:06:56.454388528 +0200
+++ CMakeLists.txt	2013-07-18 00:46:17.282726618 +0200
@@ -0,0 +1,15 @@
+PROJECT(jsoncpp)
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
+ENABLE_LANGUAGE(CXX)
+
+SET(SOURCES src/lib_json/json_reader.cpp
+            src/lib_json/json_value.cpp
+            src/lib_json/json_writer.cpp)
+
+INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR}/include)
+
+ADD_LIBRARY(jsoncpp ${SOURCES})
+
+INSTALL(TARGETS jsoncpp DESTINATION lib) 
+INSTALL(DIRECTORY include/json DESTINATION include/jsoncpp)
